bazaarvoice/emodb
# Jackson_Jdk8_Modifier_Update

**Jira Issue**
https://bazaarvoice.atlassian.net/browse/PD-211835

**What Are We Doing Here?**
Emodb is one of the 4 repositories where the critical vulnerabilities needs to be remediated in synk. Updated Jackson-datatype jdk8 modifier.

**DOD:**
All the remediation code should be tested, reviewed and merge to mainline.
Push to QA environment and validate.

**Acceptance Criteria:**
All the Improper Input Validation Vulnerabilities should be remediated in Synk(11 in number).

**How to Test and Verify**

Check out local build :
Unit testing : mvn clean install
![image](https://github.com/bazaarvoice/emodb/assets/134580247/47a84bdd-a23c-4057-bcde-55fe87c2b239)


**Risk**
Level : Low

**Required Testing**
Manual.

**Risk Summary**
Since risk level will be low,we will be monitoring for vulnerability issues of jackson library in snyk dashboard.
